### DISCUSSION TITLE
Sponsored Brands API call /v2/hsa/campaigns/report

### DISCUSSION THREAD

**Discussion by rafaldudek** [amzn/ads-advanced-tools-docs#30] [Q&A]
Hi,

I'm getting error 400 - KDP authors do not have access to Sponsored Brands functionality 

Other types of request work (/v2/sp/campaigns/report, /sd/campaigns/report), but not /v2/hsa/campaigns/report

I'm getting the same error message in Postman and PHP via CURL

What can be the issue/how to resolve it?

Thank you in advance for any help,
Regards,
Rafal

**Comment by elizafarley** [comment]
Hi @rafaldudek, while KDP authors do have access to create Sponsored Brands campaigns, some SB endpoints (including the endpoint POST v2/hsa/{recordType}/report) are not currently available.

For now, you should use the advertising console to review reporting data for KDP authors using SB.

**Comment by rafaldudek** [comment]
Hi @elizafarley - thank you so much for a quick response!
Not  sure if I understand it correctly - currently there is no API/programmatic way of getting SB  campaigns performance (impressions, clicks, cost etc) data and the only way to see/get SB data is to view/download it from AMS website?
If that's the case do you know if/when this API endpoint will be available?

Thank you in advance,
Cheers,
Rafal

**Reply by elizafarley** [reply]
Yes, currently the API does not support SB reporting for KDP. I don't have any approximate dates, but when it's available, we will post it in the [release notes](https://advertising.amazon.com/API/docs/en-us/info/release-notes).

**Reply by rafaldudek** [reply]
Thank you Eliza - have a great day!
